client/main/ribbon/datatab.js

```diff
@@ -35,12 +35,12 @@ class DataTab extends RibbonTab {
             ]}),
             new RibbonSeparator(),
             new RibbonGroup({ title: _('Variables'), items: [
-                new RibbonButton({ title: _('Setup'), name: 'editVar', margin: 'large', size: 'large', shortcutKey: 's', shortcutPosition: { x: '50%', y: '90%' } }),
-                new RibbonButton({ title: _('Compute'), name: 'compute', margin: 'large', size: 'large', shortcutKey: 'q', shortcutPosition: { x: '50%', y: '90%' } }),
-                new RibbonButton({ title: _('Transform'), name: 'transform', margin: 'large', size: 'large', shortcutKey: 't', shortcutPosition: { x: '50%', y: '90%' } }),
-                new RibbonButton({ title: _('Weights'), name: 'weights', margin: 'large', size: 'large', shortcutKey: 'w', shortcutPosition: { x: '50%', y: '90%' } }),
+                new RibbonButton({ title: _('Setup'), ariaLabel: _('Setup selected variable properties'), name: 'editVar', margin: 'large', size: 'large', shortcutKey: 's', shortcutPosition: { x: '50%', y: '90%' } }),
```

Comment:
`Variable setup`? ... they may want to view, rather than change anything. i think we want nouns rather than verbs (unless it actively does something, like a button press).